GameOfLife/GameOfLife/Engine.cs

```diff
@@ -203,14 +286,15 @@ public int CountAlive(string[,] gameField)
         /// <summary>
         /// Method to count the current number of dead cells on the field.
         /// </summary>
+        /// <param name="gameField">An array of the game field cells.</param>
         /// <returns>Returns the number of dead cells currently in the gamefield array.</returns>
         public int CountDead(string[,] gameField)
```

Comment:
I just thought that it would be nice to also see the number of dead cells. CountDead method seems to be absolutely unnecessary, changed it to simple subtraction.